### PR TITLE
Bug 1210314 Settings Implementation

### PR DESCRIPTION
**Pull request by insjinger**
Changes for Share Contact
Please review it
@Ashish-Developer @pgmanutd

### CODE REVIEW

File: apps/settings/style/settings.css [hunk 1]
```diff
@@ -190,6 +194,7 @@ ul li > a.menu-item > * {
 
 #wifi-password {
   margin-top: 0;
+ margin-top: -2rem;
```
Comment (pgmanutd) [inline apps/settings/style/settings.css]: remove margin-top: 0..its not required now.

**Comment by TatianaMigacheva** [comment]
I have some general comments: 
1)  Mozilla want to integrate each commit by separate bug number. So our internal process is the same. Each PR (development branch) should be done by separate Bug Number.
All what we are delivering in scope of scrint 1 for Settings feature we are doing on branch "bug-1210314-settings" under Bug 1210314 and under one PR. 
I see that one PR https://github.com/harman-red-square/gaia/pull/20 is already open for this branch
So please clarify if PR25 is related to sprint 1 or not. 
If not so new Bug should be opened for tracking the changes.
If yes so we will plan to integrate your PRs one by one and then squash before second Mozilla review (the same manner as Home screen feature).

2) Commit Message doesn't follow template (http://stteamsite.symphonyteleca.com/redsquare/Wiki/Commit_message_template.aspx)

File: shared/style/confirm.css [hunk 2]
```diff
@@ -347,3 +347,18 @@ form[role="dialog"][data-type="confirm"] > section#device-option-unpair h1 {
   margin-left: -1.5rem;
   font-size: 2rem;
 }
+
+#enable-bluetooth-view {
+    background-color: rgba(45, 45, 45, 0.8);
```
Comment (egorlevichev) [inline shared/style/confirm.css]: you shall use 2 spaces for indentation instead of 4

File: apps/settings/style/settings.css (hunk 1 above)
Comment (egorlevichev) [inline apps/settings/style/settings.css]: and use 2 spaces for indentation

File: shared/style/headers.css [hunk 3]
```diff
@@ -178,7 +178,7 @@ section[role="region"] > header:first-child .icon-back {
 }
 
 #settings-forwd span {
-  background-image: url(headers/images/icons/forwd.png);
+  background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAeCAQAAACROWYpAAAABGdBTUEAA5uow9umHwAAAAJiS0dEAP+Hj8y/AAAAiUlEQVQ4y+2UzQ3CMAxGv6YLsEkjukdX4NgJuLMG0zACbNNWJHlcegyoOFIRUr6TZevJP7ItVUmSOHGwwxN3I85AIG7BXcZ3lCR1uhmy0zPzJJJMxeP/H09gxRciCTjn4u4T3Dw0rmb7NYzXdTXjfj0XTPtHaMlucyHYN2vzPedx+ycp/GFV7/UClnyxgBxMauEAAAAldEVYdGRhdGU6Y3JlYXRlADIwMTUtMTAtMjFUMTI6MjA6NTUrMDI6MDA7LZarAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDE1LTEwLTIxVDEyOjIwOjU1KzAyOjAwSnAuFwAAAABJRU5ErkJggg==);
```
Comment (inppandey20) [inline shared/style/headers.css]: Please remove base64

**Comment by inppandey20** [comment]
Both PR 20, 25 belong to same bug from sprint 1

File: apps/communications/contacts/locales/contacts.en-US.properties [hunk 4]
```diff
@@ -15,7 +15,7 @@ selectContact      = Select contact
 selectAll          = Select all
 deselectAll        = Deselect all
 editContact        = Edit contact
-deleteConfirmMsg   = Delete contact?
+deleteConfirmMsg   = Are you sure you want to delete contact?
```
Comment (egorlevichev) [inline apps/communications/contacts/locales/contacts.en-US.properties]: As mentioned by David in comments for initial PR of Settings to Mozilla 2.2r branch (see https://github.com/mozilla-b2g/gaia/pull/32750) you shall not change string without assigning new l10n id. If you need different text you **shall also define new l10nid** for it, otherwise localization team will not detect this change and text will remain untranslated.
One more point, as soon as you updating this localization file, probably it will be better to fix remaining David's comments related to this file (e.g. lines 115-117), since without having them fixed we won't be allowed to land this code.

File: apps/bluetooth/locales/bluetooth.en-US.properties [hunk 5]
```diff
@@ -41,7 +41,7 @@ search-for-device   = Searching for devices…
 search-device       = Search for devices
 rename-device       = Rename my device
 unnamed-device      = Unknown device
-change-phone-name   = Change phone name
+change-phone-name   = Rename My Device
```
Comment (egorlevichev) [inline apps/bluetooth/locales/bluetooth.en-US.properties]: It's not allowed to change strings without changing l10nid key. Even more, almost the same text already defined 2 lines above (see line 42: `rename-device       = Rename my device`). According to David's comment, Mozilla UX team prefer not to capitalize first letters in all words in sentence, so while strings are not 100% defined & approved by Mozilla UX it's better to stick to Mozilla's practices.
This comment also applies to changes in lines 56-58 in this file.

File: apps/bluetooth/onpair.html [hunk 6]
```diff
@@ -51,4 +50,3 @@ <h1 id="label-name"></h1>
     </form>
   </body>
 </html>
-
```
Comment (egorlevichev) [inline apps/bluetooth/onpair.html:54]: Please avoid unnecessary changes in code which you are updating.

File: apps/bluetooth/transfer.html [hunk 7]
```diff
@@ -38,11 +38,9 @@
       <section>
         <h1 data-l10n-id="confirmation">Confirmation</h1>
         <p>
-          <img id="icon" src="style/images/settings_bluetooth.png" alt="Bluetooth Icon"/>
-          <strong data-l10n-id="bluetooth">Bluetooth</strong>
           <small data-l10n-id="bluetooth-status-description">Bluetooth is disabled</small>
         </p>
-        <p data-l10n-id="turn-bluetooth-on">Do you want to turn Bluetooth on?</p>
+        <small data-l10n-id="turn-bluetooth-on">Do you want to turn Bluetooth on?</small>
```
Comment (egorlevichev) [inline apps/bluetooth/transfer.html]: This probably shall be consistent with new text used in l10n file, so please update default text together with new l10nid key

**Comment by TatianaMigacheva** [comment]
See failed build log: \runn14lsp002\shared\builds\pull-requests\gaia_pr25\build.log

File: apps/bluetooth/locales/bluetooth.en-US.properties [hunk 8]
```diff
@@ -53,9 +53,11 @@ unpair-title = Unpair a connected device?
 unpair-msg   = You are currently connected to this device. Unpairing will also disconnect from it.
 # Bluetooth :: Confirmation Dialog
 bluetooth = Bluetooth
-confirmation = Confirmation
+Confirmation = Confirmation
```
Comment (egorlevichev) [inline apps/bluetooth/locales/bluetooth.en-US.properties]: you shall not change existing l10nId, leave 'c' not capitalized.

File: apps/bluetooth/onpair.html [hunk 9]
```diff
@@ -27,20 +27,15 @@
...
         <h1 id="label-name"></h1>
-
         <p id="pair-description"></p>
-
         <span id="confirmation-method" >
           <span id="passkey"></span>
         </span>
-
-        <span id="pin-input-method" hidden>
+          <span id="pin-input-method" hidden>
```
Comment (egorlevichev) [inline apps/bluetooth/onpair.html]: wrong indentation, 2 extra spaces

File: apps/bluetooth/onpair.html [hunk 10]
```diff
@@ -27,20 +27,15 @@
...
           <input type="text" id="pin-input" size="16" maxlength="16" dir="auto" />
         </span>
-
-        <span id="passkey-input-method" hidden>
+          <span id="passkey-input-method" hidden>
```
Comment (egorlevichev) [inline apps/bluetooth/onpair.html]: wrong indentation, 2 extra spaces

**Comment by AlekoNN** [comment]
@harman-red-square/harman-reviewers

**Comment by TatianaMigacheva** [comment]
Harman-Jenkins: pr_review_failed

**1. Branch is not up-to-date with harman-red-square:bug-1210314-settings**
Please do rebase on https://github.com/harman-red-square/gaia/tree/bug-1210314-settings

**2. Lint errors:**

gjslint check:
Running gjslint...
----- FILE  :  /export/jenkins/jobs/Gaia_PR_checker/workspace/checkstyle/gaia/apps/settings/js/call.js -----
Line 61, E:0002: Missing space after ":"
Line 195, E:0002: Missing space before "{"
Line 205, E:0010: Missing semicolon at end of line
Line 1027, E:0011: Missing semicolon after function assigned to a variable
Line 1035, E:0110: Line too long (84 characters).
Line 1051, E:0110: Line too long (86 characters).
----- FILE  :  /export/jenkins/jobs/Gaia_PR_checker/workspace/checkstyle/gaia/apps/settings/js/carrier.js -----
Line 111, E:0002: Missing space before "("
Line 113, E:0002: Missing space before "("
Line 113, E:0002: Missing space before "{"
Line 133, E:0110: Line too long (83 characters).
Line 137, E:0002: Missing space before "("
Line 138, E:0002: Missing space after ","
Line 139, E:0002: Missing space after ","
Line 140, E:0002: Missing space after ","
Line 141, E:0131: Single-quoted string preferred over double-quoted string.
Line 143, E:0131: Single-quoted string preferred over double-quoted string.
Line 144, E:0131: Single-quoted string preferred over double-quoted string.
Line 145, E:0131: Single-quoted string preferred over double-quoted string.
Line 146, E:0002: Missing space after ","
Line 172, E:0002: Missing space before "{"
Line 196, E:0002: Missing space before "("
Line 201, E:0002: Missing space before "{"
Line 206, E:0002: Missing space before "("
Line 206, E:0002: Missing space before "{"
Line 242, E:0001: Extra space after "dr_expl"
Line 274, E:0001: Extra space after "dc_expl"
Line 277, E:0001: Extra space after "dc_expl"
Found 27 errors, including 1 new error, in 2 files (0 files OK).

Some of the errors reported by GJsLint may be auto-fixable using the script
fixjsstyle. Please double check any changes it makes and report any bugs. The
script can be run by executing:

fixjsstyle --disable 210,217,220,225 --custom_jsdoc_tags=prop,borrows,memberof,augments,exports,global,event,example,mixes,mixin,fires,inner,todo,access,namespace,listens,module,memberOf,property,requires,alias,returns -e apps/pdfjs/content,apps/pdfjs/test,apps/email/build,apps/email/built,apps/email/js/ext,apps/calendar/build,apps/calendar/js/ext,apps/camera/js/vendor,apps/camera/test,apps/privacy-panel/js/vendor,apps/system/camera,apps/keyboard/js/imes/jszhuyin/lib,apps/keyboard/js/imes/handwriting/hwr,apps/wappush/js/ext,apps/keyboard/js/layouts,apps/music/bower_components,apps/camera/bower_components -x build/r.js,build/csslinter.js,build/jsmin.js,apps/communications/contacts/oauth2/js/parameters.js,apps/calendar/js/presets.js,apps/email/js/alameda.js,apps/email/js/tmpl_builder.js,apps/clock/js/alameda.js,apps/clock/js/text_builder.js,shared/js/utilities.js,shared/js/uuid.js,apps/keyboard/js/imes/jspinyin/libpinyin.js,shared/js/setImmediate.js,shared/test/unit/test-agent-server.js,shared/elements/gaia-header/dist/gaia-header.js,shared/elements/gaia-component/gaia-component.js,shared/elements/gaia-icons/gaia-icons.js,shared/pages/import/js/parameters.js,apps/sharedtest/test/unit/setImmediate_test.js,apps/keyboard/js/imes/jsavrophonetic/jsavrophonetic.js,apps/camera/js/vendor/alameda.js,apps/camera/js/config.js,apps/gallery/js/config.js,apps/gallery/js/frame_scripts.js,apps/gallery/js/metadata_scripts.js,apps/music/js/metadata_scripts.js,apps/keyboard/js/imes/jszhuyin/tools/cook.js,apps/keyboard/build/keyboard-config.js,apps/settings/js/vendor/alameda.js,apps/findmydevice/js/lib/*,apps/findmydevice/js/config.js,apps/system/js/uuid.js,apps/bluetooth/js/settings.js,tests/atoms/remote_date.js,tests/atoms/screenshot.js,tests/performance/getappname_atom.js,tests/performance/exec-sync.js apps/settings/js/call.js apps/settings/js/carrier.js 

jshint check:
apps/settings/js/call.js: jshint errors are ignored. Please consider fixing lint errors and removing it from build/jshint/xfail.list.
apps/settings/js/carrier.js: jshint errors are ignored. Please consider fixing lint errors and removing it from build/jshint/xfail.list.
apps/communications/contacts/js/views/ice_settings.js: line 201, col 85, Line is too long. (ERROR)
apps/communications/contacts/js/views/ice_settings.js: line 201, col 47, Strings must use singlequote. (ERROR)
apps/communications/contacts/js/views/ice_settings.js: line 201, col 85, Strings must use singlequote. (ERROR)
apps/communications/contacts/js/views/ice_settings.js: line 36, col 29, '\''navigationStack'\'' is not defined. (ERROR)
apps/communications/contacts/js/views/list.js: line 530, col 51, Strings must use singlequote. (ERROR)
apps/communications/contacts/js/views/list.js: line 531, col 35, Strings must use singlequote. (ERROR)
apps/communications/contacts/js/views/list.js: line 1677, col 79, Strings must use singlequote. (ERROR)
apps/communications/contacts/js/views/list.js: line 1699, col 31, Don'\''t make functions within a loop. (ERROR)
apps/communications/contacts/js/views/list.js: line 1681, col 13, '\''OptionMenu'\'' is defined but never used. (ERROR)
apps/settings/js/findmydevice.js: line 159, col 43, Strings must use singlequote. (ERROR)
apps/settings/js/findmydevice.js: line 161, col 47, Strings must use singlequote. (ERROR)
apps/settings/js/findmydevice.js: line 165, col 44, Strings must use singlequote. (ERROR)
apps/settings/js/findmydevice.js: line 166, col 45, Strings must use singlequote. (ERROR)
apps/settings/js/findmydevice.js: line 170, col 47, Strings must use singlequote. (ERROR)
apps/settings/js/findmydevice.js: line 171, col 49, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 88, col 34, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 109, col 30, '\''onVisibilityChange'\'' is already defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 193, col 49, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 198, col 48, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 203, col 48, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 211, col 55, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 36, col 5, '\''menuStatus1'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 37, col 5, '\''menuStatus'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 91, col 7, '\''menuStatus'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 92, col 7, '\''menuStatus'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 93, col 39, '\''menuStatus1'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 96, col 39, '\''menuStatus'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 99, col 41, '\''menuStatus1'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 103, col 39, '\''menuStatus'\'' is not defined. (ERROR)
apps/settings/js/panels/browsing_privacy/panel.js: line 31, col 86, Line is too long. (ERROR)
apps/settings/js/panels/browsing_privacy/panel.js: line 40, col 85, Line is too long. (ERROR)
apps/settings/js/panels/improveBrowserOS/panel.js: line 14, col 82, Line is too long. (ERROR)
apps/settings/js/panels/improveBrowserOS/panel.js: line 12, col 9, '\''SettingsListener'\'' is not defined. (ERROR)
apps/settings/js/panels/root/panel.js: line 68, col 85, Line is too long. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 34, col 86, Line is too long. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 221, col 57, Strings must use singlequote. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 222, col 48, Strings must use singlequote. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 222, col 73, Strings must use singlequote. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 239, col 58, Strings must use singlequote. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 240, col 48, Strings must use singlequote. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 240, col 74, Strings must use singlequote. (ERROR)

41 errors (71 xfailed)
There were errors while linting the files, please see above.
Please read https://github.com/mozilla-b2g/gaia/tree/master/build/jshint/README.md for more information.

**Comment by TatianaMigacheva** [comment]
Harman-Jenkins: pr_review_failed

**1. Branch is not up-to-date with harman-red-square:bug-1210314-settings**
Please do rebase on https://github.com/harman-red-square/gaia/tree/bug-1210314-settings

**2. Lint errors:**

gjslint check:
Running gjslint...
----- FILE  :  /export/jenkins/jobs/Gaia_PR_checker/workspace/checkstyle/gaia/apps/settings/js/call.js -----
Line 61, E:0002: Missing space after ":"
Line 195, E:0002: Missing space before "{"
Line 205, E:0010: Missing semicolon at end of line
Line 1027, E:0011: Missing semicolon after function assigned to a variable
Line 1035, E:0110: Line too long (84 characters).
Line 1051, E:0110: Line too long (86 characters).
----- FILE  :  /export/jenkins/jobs/Gaia_PR_checker/workspace/checkstyle/gaia/apps/settings/js/carrier.js -----
Line 111, E:0002: Missing space before "("
Line 113, E:0002: Missing space before "("
Line 113, E:0002: Missing space before "{"
Line 133, E:0110: Line too long (83 characters).
Line 137, E:0002: Missing space before "("
Line 138, E:0002: Missing space after ","
Line 139, E:0002: Missing space after ","
Line 140, E:0002: Missing space after ","
Line 141, E:0131: Single-quoted string preferred over double-quoted string.
Line 143, E:0131: Single-quoted string preferred over double-quoted string.
Line 144, E:0131: Single-quoted string preferred over double-quoted string.
Line 145, E:0131: Single-quoted string preferred over double-quoted string.
Line 146, E:0002: Missing space after ","
Line 172, E:0002: Missing space before "{"
Line 196, E:0002: Missing space before "("
Line 201, E:0002: Missing space before "{"
Line 206, E:0002: Missing space before "("
Line 206, E:0002: Missing space before "{"
Line 242, E:0001: Extra space after "dr_expl"
Line 274, E:0001: Extra space after "dc_expl"
Line 277, E:0001: Extra space after "dc_expl"
Found 27 errors, including 1 new error, in 2 files (0 files OK).

Some of the errors reported by GJsLint may be auto-fixable using the script
fixjsstyle. Please double check any changes it makes and report any bugs. The
script can be run by executing:

fixjsstyle --disable 210,217,220,225 --custom_jsdoc_tags=prop,borrows,memberof,augments,exports,global,event,example,mixes,mixin,fires,inner,todo,access,namespace,listens,module,memberOf,property,requires,alias,returns -e apps/pdfjs/content,apps/pdfjs/test,apps/email/build,apps/email/built,apps/email/js/ext,apps/calendar/build,apps/calendar/js/ext,apps/camera/js/vendor,apps/camera/test,apps/privacy-panel/js/vendor,apps/system/camera,apps/keyboard/js/imes/jszhuyin/lib,apps/keyboard/js/imes/handwriting/hwr,apps/wappush/js/ext,apps/keyboard/js/layouts,apps/music/bower_components,apps/camera/bower_components -x build/r.js,build/csslinter.js,build/jsmin.js,apps/communications/contacts/oauth2/js/parameters.js,apps/calendar/js/presets.js,apps/email/js/alameda.js,apps/email/js/tmpl_builder.js,apps/clock/js/alameda.js,apps/clock/js/text_builder.js,shared/js/utilities.js,shared/js/uuid.js,apps/keyboard/js/imes/jspinyin/libpinyin.js,shared/js/setImmediate.js,shared/test/unit/test-agent-server.js,shared/elements/gaia-header/dist/gaia-header.js,shared/elements/gaia-component/gaia-component.js,shared/elements/gaia-icons/gaia-icons.js,shared/pages/import/js/parameters.js,apps/sharedtest/test/unit/setImmediate_test.js,apps/keyboard/js/imes/jsavrophonetic/jsavrophonetic.js,apps/camera/js/vendor/alameda.js,apps/camera/js/config.js,apps/gallery/js/config.js,apps/gallery/js/frame_scripts.js,apps/gallery/js/metadata_scripts.js,apps/music/js/metadata_scripts.js,apps/keyboard/js/imes/jszhuyin/tools/cook.js,apps/keyboard/build/keyboard-config.js,apps/settings/js/vendor/alameda.js,apps/findmydevice/js/lib/*,apps/findmydevice/js/config.js,apps/system/js/uuid.js,apps/bluetooth/js/settings.js,tests/atoms/remote_date.js,tests/atoms/screenshot.js,tests/performance/getappname_atom.js,tests/performance/exec-sync.js apps/settings/js/call.js apps/settings/js/carrier.js 

jshint check:
apps/settings/js/call.js: jshint errors are ignored. Please consider fixing lint errors and removing it from build/jshint/xfail.list.
apps/settings/js/carrier.js: jshint errors are ignored. Please consider fixing lint errors and removing it from build/jshint/xfail.list.
apps/communications/contacts/js/views/ice_settings.js: line 201, col 85, Line is too long. (ERROR)
apps/communications/contacts/js/views/ice_settings.js: line 201, col 47, Strings must use singlequote. (ERROR)
apps/communications/contacts/js/views/ice_settings.js: line 201, col 85, Strings must use singlequote. (ERROR)
apps/communications/contacts/js/views/ice_settings.js: line 36, col 29, '\''navigationStack'\'' is not defined. (ERROR)
apps/communications/contacts/js/views/list.js: line 530, col 51, Strings must use singlequote. (ERROR)
apps/communications/contacts/js/views/list.js: line 531, col 35, Strings must use singlequote. (ERROR)
apps/communications/contacts/js/views/list.js: line 1677, col 79, Strings must use singlequote. (ERROR)
apps/communications/contacts/js/views/list.js: line 1699, col 31, Don'\''t make functions within a loop. (ERROR)
apps/communications/contacts/js/views/list.js: line 1681, col 13, '\''OptionMenu'\'' is defined but never used. (ERROR)
apps/settings/js/findmydevice.js: line 159, col 43, Strings must use singlequote. (ERROR)
apps/settings/js/findmydevice.js: line 161, col 47, Strings must use singlequote. (ERROR)
apps/settings/js/findmydevice.js: line 165, col 44, Strings must use singlequote. (ERROR)
apps/settings/js/findmydevice.js: line 166, col 45, Strings must use singlequote. (ERROR)
apps/settings/js/findmydevice.js: line 170, col 47, Strings must use singlequote. (ERROR)
apps/settings/js/findmydevice.js: line 171, col 49, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 88, col 34, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 109, col 30, '\''onVisibilityChange'\'' is already defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 193, col 49, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 198, col 48, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 203, col 48, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 211, col 55, Strings must use singlequote. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 36, col 5, '\''menuStatus1'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 37, col 5, '\''menuStatus'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 91, col 7, '\''menuStatus'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 92, col 7, '\''menuStatus'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 93, col 39, '\''menuStatus1'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 96, col 39, '\''menuStatus'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 99, col 41, '\''menuStatus1'\'' is not defined. (ERROR)
apps/settings/js/firefox_accounts/panel.js: line 103, col 39, '\''menuStatus'\'' is not defined. (ERROR)
apps/settings/js/panels/browsing_privacy/panel.js: line 31, col 86, Line is too long. (ERROR)
apps/settings/js/panels/browsing_privacy/panel.js: line 40, col 85, Line is too long. (ERROR)
apps/settings/js/panels/improveBrowserOS/panel.js: line 14, col 82, Line is too long. (ERROR)
apps/settings/js/panels/improveBrowserOS/panel.js: line 12, col 9, '\''SettingsListener'\'' is not defined. (ERROR)
apps/settings/js/panels/root/panel.js: line 68, col 85, Line is too long. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 34, col 86, Line is too long. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 221, col 57, Strings must use singlequote. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 222, col 48, Strings must use singlequote. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 222, col 73, Strings must use singlequote. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 239, col 58, Strings must use singlequote. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 240, col 48, Strings must use singlequote. (ERROR)
apps/settings/js/panels/wifi/panel.js: line 240, col 74, Strings must use singlequote. (ERROR)

41 errors (71 xfailed)
There were errors while linting the files, please see above.
Please read https://github.com/mozilla-b2g/gaia/tree/master/build/jshint/README.md for more information.

**Comment by TatianaMigacheva** [comment]
closed, as PR#41 has been created instead of PR#25
